Jbarget/oxymore
# creates fashion editorial page

### What changes have you made?
- reduced all image files to 1 MB or below
- added all images to a scrollable grid 

### Which issue(s) does this PR fix?

Fixes #14 

### Screenshots (if there are design changes)
<img width="232" alt="Screenshot 2020-11-08 at 19 59 40" src="https://user-images.githubusercontent.com/53219789/98484110-85d24000-220d-11eb-8ac8-03672221d5cf.png">
<img width="725" alt="Screenshot 2020-11-08 at 20 00 10" src="https://user-images.githubusercontent.com/53219789/98484147-b87c3880-220d-11eb-855a-8e1a6f2ef79b.png">


### How to test
nav to http://localhost:3000/projects/editorial and check the grid scroll on desktop and mobile
